packages/ui/src/components/shadcn/ui/sonner.tsx

```diff
@@ -48,8 +53,21 @@ const SonnerToaster = ({ ...props }: ToasterProps) => {
             'group toast group-[.toaster]:!bg-warning-200 group-[.toaster]:!border-warning-500',
           error:
             'group toast group-[.toaster]:!bg-destructive-200 group-[.toaster]:!border-destructive-500',
+          closeButton: cn(
+            // unset all styles set from sonner
+            'absolute right-2 top-2 rounded-md text-foreground/50 opacity-0 transition-opacity',
+            'hover:text-foreground focus:opacity-100 focus:outline-none focus:ring-2 group-hover:opacity-100',
+            'group-[.destructive]:text-red-300 group-[.destructive]:hover:text-red-50 group-[.destructive]:focus:ring-red-400',
+            'group-[.destructive]:focus:ring-offset-red-600',
+            'left-auto transform-none bg-transparent border-0 border-transparent hover:!bg-transparent hover:border-transparent'
+          ),
+          content: 'grow',
+          //group-[.toaster]:bg-overlay group-[.toaster]:text-foreground group-[.toaster]:border group-[.toaster]:border-overlay
         },
+        closeButton: true,
```

Comment:
why is there a close button by default ?

Reply:
Toasts in studio on production already have close buttons. I'm just matching the current implementation. Don't have a strong opinion on this.